docs/tokens.md

```diff
@@ -30,11 +36,19 @@ TxData {
 }
 ```
 
-## Issue Tokens
-When issuing a new token, we specify the data for creating a new token in the transaction input, where the `token_id` is a hash of the inputs. **TODO which inputs?**
-**TODO explain remaining fields**
+## Issuing Tokens
+
+**TODO is the sentence below true? In the example, I don't see any data relating to the token in the input.** 
+
+When issuing a new token, we specify the data for creating a new token in the transaction input, where the `token_id` is a hash of the inputs.
+
+**TODO explain the remaining fields (token_ticker, metadata_URI)**
+
+**TODO the token issuance struct shown here has a token_id which does not appear in the example. In staging, this is commented out. Why?**
```

Comment:
During the review process, Sam tell me that I have to remove token_id in TokenIssuanceV1 because it's a calculatable field from an of the first input. I will clear comments.